wiki/Contests/Skinning_Contest/1/en.md

```diff
@@ -0,0 +1,139 @@
+---
+tags:
+  - skinning
+  - skinship
+  - skins
+---
+
+# Skinning Contest #1
+
+![Skinning Contest #1 Banner](img/banner.jpg)
+
+The **Skinning Contest #1** is a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz/ "skinship"), one of the largest skinning communities on osu!. This is the first official osu! skinning contest, and the initial iteration of Skinning Contests hosted by skinship.
+
+## Contest schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Announcement | 2021-03-08 |
+| Submission phase | 2021-03-08/2021-04-08 |
+| Voting phase | 2021-04-29/2021-05-13 |
+| Results | 2021-05-20 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, unique profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu! supporter, unique profile badge |
+
+## Organisation
+
+The Skinning Contest is run by various community members.
+
+| Member | Responsibility |
+| :-- | :-- |
+| ![][flag_DE] [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Contest planning |
+| ![][flag_RO] [Matt2e2](https://osu.ppy.sh/users/12144912) | Contest planning, graphic design |
+| ![][flag_NL] [Roan](https://osu.ppy.sh/users/8214639) | Contest planning |
+| ![][flag_DE] [RockRoller](https://osu.ppy.sh/users/8388854) | Contest planning, submission screening, showcase preparations |
+| ![][flag_GB] [tetsui](https://osu.ppy.sh/users/10974678) | Contest planning, showcase preparations |
+| ![][flag_PL] [watterino](https://osu.ppy.sh/users/3512261) | Video editor, showcase preparations |
+| ![][flag_NL] [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
+
+## Links
+
+- [**Contest page**](https://osu.ppy.sh/community/contests/126)
+- [Announcement newspost](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)
+- [Voting Open newspost](https://osu.ppy.sh/home/news/2021-04-29-skinning-contest-voting-open)
+- [Results Out newspost](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results)
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1266342)
+- [Submission thread](https://osu.ppy.sh/community/forums/topics/1266324)
+- [Discord server](https://discord.skinship.xyz/)
+- [Official website](https://skinship.xyz/)
```

Comment:
```suggestion
- [Discord server](https://discord.gg/q8Z5JXM "Discord")
- [Official website](https://skinship.xyz "skinship")
```

Reply:
Don't see the point of this one? a domain based invite is more likely to survive over time than a direct one. The discord one may change but the domain one will always lead to the server